drivers/NFSSR.py

```diff
@@ -273,6 +274,7 @@ def generate_config(self, sr_uuid, vdi_uuid):
         resp['sr_uuid'] = sr_uuid
         resp['vdi_uuid'] = vdi_uuid
         resp['sr_sm_config'] = self.sr.sm_config
+        resp['sr_other_config'] = self.sr.other_config
```

Comment:
Why is other_config a requisite for NFSSR attach? Also, this throws a run-time exception. There is no self.sr.other_config